scripts/build.sh

```diff
@@ -1,13 +1,24 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
 
-SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 REPO_BASE_DIR=${SCRIPTS_DIR}/..
 
-echo "Building the universe!";
+echo "Building the universe!"
 
-mkdir -p ${REPO_BASE_DIR}/target/;
+mkdir -p ${REPO_BASE_DIR}/target/
 
-$SCRIPTS_DIR/"0-validate-version.sh";
-$SCRIPTS_DIR/"validate-packages.py";
-$SCRIPTS_DIR/gen-universe.py --repository=${REPO_BASE_DIR}/repo/packages/ --out-dir=${REPO_BASE_DIR}/target/;
+# Create a new virtual environment
+rm -rf ${REPO_BASE_DIR}/target/venv
+python3 -m venv ${REPO_BASE_DIR}/target/venv
+
+# Install dependencies
+${REPO_BASE_DIR}/target/venv/bin/pip install -r ${SCRIPTS_DIR}/requirements/requirements.txt
```

Comment:
The important change is here. 1) create a virtual env and 2) install the requirements.